### PR TITLE
SFR-2099: Edge Case API Tests for get/collection

### PR DESCRIPTION
**Pull request by ayan1229**
This test validates the edge cases for get/collection API

 [SFR-2299](https://newyorkpubliclibrary.atlassian.net/browse/SFR-2299)

[SFR-2299]: https://newyorkpubliclibrary.atlassian.net/browse/SFR-2299?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: tests/integration/api/test_get_collection.py [hunk 1]
```diff
@@ -13,3 +13,23 @@ def test_get_collection():
...
+
+def test_get_collection_empty_id():
+    url = API_URL + '/collection/'
+    response = requests.get(url)
+    assert_response_status(url, response, 404)
+
+def test_get_collection_special_characters():
+    url = API_URL + '/collection/%$@!*'
+    response = requests.get(url)
+    assert_response_status(url, response, 400)
```
Comment (kylevillegas93) [inline tests/integration/api/test_get_collection.py:35]: nit: add new-line!

**Review by kylevillegas93 — APPROVED** [review]
This looks good!
